wp-content/themes/missilethreat/inc/template-tags.php

```diff
@@ -334,12 +334,13 @@ function missiledefense_display_system_elements() {
 				$archiveTitle = esc_html__( 'System Elements', 'missiledefense' );
 			}
 
-			$html = '<div class="system-elements">
-					<h1>' . $archiveTitle . '</h1>
-					<ul role="list">';
+			$html = '<div class="system-elements alignright">
```

Comment:
In this function before you do the `wp_get_post_terms`, I would do a check to see if the post type matches defsys, so this function isn't unnecessarily running on regular posts or other post types.